lib/httpOperationResponse.ts

```diff
@@ -5,10 +5,9 @@ import { WebResource } from "./webResource";
 import { HttpHeaders } from "./httpHeaders";
 
 /**
- * Wrapper object for http request and response. Deserialized object is stored in
- * the `parsedBody` property when the response body is received in JSON or XML.
+ * The properties on an HTTP response which will always be present.
  */
-export interface HttpOperationResponse<TBody = any, THeaders = any> {
+export interface BaseHttpOperationResponse {
```

Comment:
We previously thought about renaming HttpOperationResponse to HttpResponse. Would this be a good time to do that, rather that adding the "Base" prefix?

Reply:
So you think HttpResponse as the base, HttpOperationResponse as the "bucket of all the optionals"?

Reply:
Yeah. Your design seems to say that HttpOperationResponse is being kept around just for backwards compatability and that BaseHttpOperationResponse/HttpResponse is what should be used/extended moving forward.